inductiva/tasks/task.py

```diff
@@ -278,6 +278,46 @@ class Task:
         models.TaskStatusCode.PENDINGINPUT, models.TaskStatusCode.STARTED
     }
 
+    STATUS_MESSAGES = {
+        models.TaskStatusCode.PENDINGINPUT:
+            "Task %s was created and is waiting for the required input files "
+            "to be uploaded.",
+        models.TaskStatusCode.EXECUTERTERMINATED:
+            "The machine running your task was terminated by the cloud provider"
+            " and your task stopped running.\nUse the `resubmit_on_preemption`"
```

Comment:
`running` and `task` repeated twice in the same sentence

Reply:
Is this better ?
```
            "The machine running your task was terminated by the cloud provider"
            " and it stopped.\nUse the `resubmit_on_preemption` parameter in "
            "the Python client to configure if the task should be automatically"
            " resubmitted to the queue.",
```